packages/reactstrap-validation-select/src/AvSelectField.js

```diff
@@ -41,9 +41,12 @@ class AvSelectField extends Component {
 
     const validation = this.context.FormCtrl ? this.context.FormCtrl.getInputState(name) : {};
     let feedback = null;
+    let ariaFeedback = {};
     if (validation.errorMessage) {
       const feedbackClasses = classNames('d-block', feedbackClass);
-      feedback = <AvFeedback className={feedbackClasses}>{validation.errorMessage}</AvFeedback>;
+      const feedbackId = `${id}-feedback`;
+      feedback = <AvFeedback className={feedbackClasses} id={feedbackId}>{validation.errorMessage}</AvFeedback>;
+      ariaFeedback = {hasError: true, ariaErrorId: feedbackId, ariaErrorDesc: validation.errorMessage}
```

Comment:
Why are we hardcoding hasError to true? feels like it defeats the purpose of having a variable. the logic for touched and invalid should already be available in avSelect that you can use instead of passing a boolean. 

Also, is there a reason why the names of the variables are constantly changing? for consistency I would rename ariaErrorId as feedbackId, ariaErrorDesc to errorMessage, and pick whether you want ariaFeedback or ariaMessage and keep it consistent between both AvSelect and AvSelectField